src/AutomatedCar/Views/CourseDisplayView.xaml

```diff
@@ -48,11 +48,10 @@
 							<TransformGroup>
 								<RotateTransform  Angle="{Binding Angle}" />
 								<TranslateTransform  X="{Binding RotationCenterPointX}" />
-								<TranslateTransform  Y="{Binding RotationCenterPointY}" />
-							</TransformGroup>
-							
+								<!-- <TranslateTransform  Y="{Binding RotationCenterPointY}" /> -->
```

Comment:
`Y` ki van kommentelve igy is müködik? ha igen akkor kell a kódba?

Reply:
Köszi. Javítottam.